chromium_src/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc

```diff
@@ -149,6 +149,8 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
       IDS_SETTINGS_IPFS_COMPANION_ENABLED_DESC},
     {"mediaRouterEnabledDesc",
       IDS_SETTINGS_MEDIA_ROUTER_ENABLED_DESC},
+    {"torDisabledLabel",
```

Comment:
same as above disabled -> enabled. It is almost always better to use the "positive" form

Reply:
done.